CHANGELOG/CHANGELOG-0.3.md

```diff
@@ -0,0 +1,25 @@
+# Changelog 0.3
+
+## 0.3.0 - 2022-10-19
+
+### Added
+
+- Theia Manager. ([#97](https://github.com/antrea-io/theia/pull/97), [@wsquan171]) ([#109](https://github.com/antrea-io/theia/pull/109), [@wsquan171]) ([#111](https://github.com/antrea-io/theia/pull/111), [@yuntanghsu]) ([#113](https://github.com/antrea-io/theia/pull/113), [@wsquan171]) ([#114](https://github.com/antrea-io/theia/pull/114), [@yanjunz97]) ([#115](https://github.com/antrea-io/theia/pull/115), [@yuntanghsu]) 
+  * Introduce a new component Theia Manager to implement the control logics
+  * Update policy recommendation and CLIs to use Theia Manager 
+- Enhanced UI. ([#101](https://github.com/antrea-io/theia/pull/101), [@heanlan]) ([#102](https://github.com/antrea-io/theia/pull/102), [@yuntanghsu])
+  * Introduce a Grafana home dashboard
+  * Improvement for throughput related panels in Grafana
+- Support for Snowflake. ([#112](https://github.com/antrea-io/theia/pull/112), [@antoninbas]) ([#118](https://github.com/antrea-io/theia/pull/118), [@heanlan]) ([#119](https://github.com/antrea-io/theia/pull/119), [@heanlan])
+  * Introduce Snowflake backend for Theia
+  * Enhance UI support for Snowflake
+
+### Changed
+
+- Adopt Golang version ClickHouse data schema management ([#95](https://github.com/antrea-io/theia/pull/95), [@yanjunz97])
```

Comment:
please change this to something like:

`Adopt golang-migrate` for ClickHouse data schema management